README.md

```diff
@@ -2,92 +2,82 @@
 
 [![Build Status](https://travis-ci.org/AugurProject/augur-core.svg)](https://travis-ci.org/AugurProject/augur-core)
 
-Ethereum contracts for a decentralized prediction market platform.
+Smart contracts for [Augur](https://augur.net), a decentralized prediction market platform on the [Ethereum](https://ethereum.org) blockchain.
 
 ## Installation
 
-You should already have a system-wide installation of Python and it should be Python 2.7.6 or greater.
+You need a system-wide installation of Python (v2.7.6 or higher).  Install the dependencies:
 
-First install the dependencies, which include PyEthereum (the tool used to test Ethereum smart contracts from Python scripts) and the Solidity smart contract programming language:
-
-```
+```bash
 npm install npx
 npm install
-
-sudo pip install -r requirements.txt
+pip install -r requirements.txt
 ```
 
-For PyEthereum to work properly you'll also need to install the 0.4.18 Solc compiler on your machine. Instructions for installing it in your environment can be found here:
+The dependencies include [pyethereum](https://github.com/ethereum/pyethereum), which is used to unit test our Ethereum smart contracts.  If you want to run the the augur-node tests locally, you also need to install [version 0.4.18 of the Solidity compiler](https://github.com/ethereum/solidity/releases/tag/v0.4.18) (`solc`) on your machine for pyethereum to run correctly.  (On macOS, you need to use [virtualenv](https://python-guide-pt-br.readthedocs.io/en/latest/dev/virtualenvs/) or [homebrew](https://brew.sh/) Python to work around System Integrity Protection.)
```

Comment:
"If you want to run the augur-node tests locally": The only other action you can take on this repo is to deploy, and you should _NOT_ be deploying locally and if you do you should not do so without running the tests first.  I recommend deleting a good chunk of this section and just including Solidity in the dependency set above (see other comment).  Also saying what is in `requirements.txt` seems a bit unnecessary, if the person wants to know what the PIP dependencies are they can look at `requirements.txt`, just like they can look at `package.json` if they want to know about our NPM dependencies.  The bit about virtualenv or homebrew to get things working on MacOS is valuable if it is non-obvious (not familiar with Mac/Python).